flex/error_messages.py

```diff
@@ -26,6 +26,7 @@
 FORMAT_MESSAGES = {
     'invalid': "Value {0} does not conform to the format {1}",
     'invalid_uuid': "{0} is not a valid uuid",
+    'invalid_date': "{0} is not a valid RFC3339 full-date",
```

Comment:
Perhaps this should say IS8601 date, - RFC339 is what Swagger says but inside there it's refering to ISO8601 in any case